### ISSUE TITLE
(IR) catalog_controller#update incoming version greater than 1, but object does not exist

### ISSUE THREAD

**Issue by SaravShah** [sul-dlss/preservation_catalog#720]
If the object was never created, the way our logic is that, if the incoming_version is greater than 1 always update. 
- You can't update an object that was never created 

for example:
```
I, [2018-05-02T10:31:21.210931 #6023]  INFO -- : [072d651e-4821-4ede-883d-d12dd775ab74] Started PATCH "/catalog/druid:td539fp3125" for 172.20.206.158 at 2018-05-02 10:31:21 -0700
I, [2018-05-02T10:31:21.216081 #6023]  INFO -- : [072d651e-4821-4ede-883d-d12dd775ab74] Processing by CatalogController#update as */*
I, [2018-05-02T10:31:21.216278 #6023]  INFO -- : [072d651e-4821-4ede-883d-d12dd775ab74]   Parameters: {"druid"=>"druid:td539fp3125", "incoming_size"=>"2052404175", "incoming_version"=>"3", "storage_location"=>"/services-disk14"}
W, [2018-05-02T10:31:21.271376 #6023]  WARN -- : [072d651e-4821-4ede-883d-d12dd775ab74] update_version(druid:td539fp3125, services-disk14) #<ActiveRecord::RecordNotFound: Couldn't find PreservedObject> db object does not exist
I, [2018-05-02T10:31:21.273753 #6023]  INFO -- : [072d651e-4821-4ede-883d-d12dd775ab74] Completed 404 Not Found in 57ms (Views: 0.6ms | ActiveRecord: 16.4ms)
```

**Comment by ndushay** [comment]
So the question raised here is: should a ReST request to *update* an object (because the object version is greater than 1) ... create the object if it doesn't exist?

@julianmorley what say you?

We encountered this situation when we put the new preservation robots into production -- if the object gets to version 2 or higher before we have an entry in the catalog, the robot code sends an UPDATE request ...  

on the one hand:  it seems correct to error -- like - hey!  create the record first!

on the other hand:  we will likely encounter this from time to time, esp while we don't have M2C running on a cron, but even after that.

Side Note:  We *could* set up M2C and C2M to run on crons now, if we cared to ...  we have the capability.
